src/ekomark/benchmark/external/LHA_utils.py

```diff
@@ -129,17 +129,17 @@ def compute_LHA_data(
         ref : dict
             output containing: target_xgrid, values
     """
-
+    Polarized= operators["polarized"]
```

Comment:
Python convention is `variables_are_all_snake_case`

Reply:
PascalCase is reserved for classes (and type hints, which overlap for a large part)

Reply:
Even if "outdated", this is still present, and causing wrong highlight in code snippets...